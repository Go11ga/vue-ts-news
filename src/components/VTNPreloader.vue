<template>
    <div class="preloader">
        <img class="preloader__img" src="@/assets/images/preloader.gif" alt="preloader">
    </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
    name: 'VTNPreloader'
})
</script>
