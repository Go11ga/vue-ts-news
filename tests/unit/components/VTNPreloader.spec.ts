import { shallowMount } from '@vue/test-utils'
import VTNPreloader from '@/components/VTNPreloader.vue'

describe('VTNPreloader', () => {
    let wrapper = shallowMount(VTNPreloader)

    it('dummy name test', () => {
        expect(wrapper.vm.$options.name).toMatch('VTNPreloader')
    })

    it('snapshot', () => {
        expect(wrapper.element).toMatchSnapshot()
    })
})
