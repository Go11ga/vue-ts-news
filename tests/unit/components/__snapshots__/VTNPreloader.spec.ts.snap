// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTNPreloader snapshot 1`] = `
<div
  class="preloader"
>
  <img
    alt="preloader"
    class="preloader__img"
  />
</div>
`;
